contracts/linear/src/validator_pool.rs

```diff
@@ -865,10 +865,14 @@ impl Validator {
         staked_balance: Balance,
         unstaked_balance: Balance,
     ) {
-        // allow at most 1 yN diff in total balance
+        // allow at most 1 NEAR diff in total balance
         let new_total_balance = staked_balance + unstaked_balance;
         require!(
-            abs_diff_eq(new_total_balance, self.total_balance(), 1),
+            abs_diff_eq(
+                new_total_balance,
+                self.total_balance(),
+                MAX_SYNC_BALANCE_DIFF
+            ),
             format!(
                 "{}. new: {}, old: {}",
                 ERR_SYNC_BALANCE_BAD_TOTAL,
```

Comment:
The error message `ERR_SYNC_BALANCE_BAD_TOTAL` needs to be updated.

`Diff between new total balance and old balance > 1 yN`